crypto/secp256k1/libsecp256k1/src/ecmult_impl.h

```diff
@@ -290,7 +290,7 @@ static void secp256k1_ecmult(const secp256k1_ecmult_context *ctx, secp256k1_gej
 #ifdef USE_ENDOMORPHISM
     secp256k1_ge pre_a_lam[ECMULT_TABLE_SIZE(WINDOW_A)];
     secp256k1_scalar na_1, na_lam;
-    /* Splitted G factors. */
+    /* Split G factors. */
```

Comment:
please revert, this file is imported from upstream. Better to avoid changing it locally